Blockstack/Classes/BlockstackConstants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  BlockstackConstants.swift
+//  Blockstack
+//
+//  Created by Michal Ciurus on 02/04/2018.
+//
+
+import Foundation
+
+enum BlockstackConstants {
```

Comment:
I suggest using `enum` for constants as enums [can't be accidentally instantiated](https://stackoverflow.com/a/38585994/2596099).